README.md

```
@@ -10,6 +10,12 @@ _Frontend for Aetheryte API_
 
 - Node `20.3.0`
 
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)